### DISCUSSION TITLE
Command suggestion seems doesn't work.

### DISCUSSION THREAD

**Discussion by Tgist1024** [ayamir/nvimdots#520] [Q&A]
![image](https://user-images.githubusercontent.com/101801875/218802781-be73553f-2e07-4331-863d-f7b8810d30c7.png)
Don't know if it's an upstream issue or an existing problem after pulling the new change or any else, so I put it in here.

**Comment by charliie-dev** [comment]
I had similar problems before, it's probably lack of certain third-party tools or unsuccessful plugin installation. Normally I just delete the entire plugin folder and let lazy reinstall everything.

**Reply by Tgist1024** [reply]
Tried, but didn't work. I even deleted `~/.config/nvim` and `~/.local/share/nvim` and then reinstalled the whole project without personal config, still useless.
Thanks for your advice, maybe I should open an issue.

**Reply by charliie-dev** [reply]
yeah, in this case probably. It's a bit weird tho.
I think we need some more info to diagnose the issue.

**Reply by Tgist1024** [reply]
I think so.
The issue opened, and if any more info is needed, please tell me.

**Reply by charliie-dev** [reply]
cc #521

**Reply by charliie-dev** [reply]
one possible fix is to remove wilder and fzy-native from plugin folder and reinstall it.

**Reply by Tgist1024** [reply]
Tried both removing by lazy.nvim and deleting the folder, didn't work.😭

**Reply by aarnphm** [reply]
might also try to remove `~/.cache/nvim` and `~/.local/state/nvim`

**Reply by Tgist1024** [reply]
thanks, but the same.

**Comment by Tgist1024** [comment]
https://github.com/ayamir/nvimdots/issues/521#issuecomment-1430983315
